### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by murphydanger**
Addresses Issue #124 

@shinnn merge whenever you read this and are happy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ var minifyCss = require('gulp-minify-css');
 
 ### minifyCss([*options*])
 
-*options*: `Object`  
+*options*: `Object` (see [clean-css](https://github.com/jakubpawlowicz/clean-css#how-to-use-clean-css-api) for more info)
+
 Return: `Object` ([stream.Transform](https://nodejs.org/docs/latest/api/stream.html#stream_class_stream_transform))
 
 Options are directly passed to the [`CleanCSS` constructor](https://github.com/jakubpawlowicz/clean-css#how-to-use-clean-css-programmatically) so all the clean-css options are available.
```
Comment (shinnn) [inline README.md]: IMHO we'd rather should fix this outdated "`CleanCSS` constructor" link than add almost duplicated info to L32.

Comment (murphydanger) [inline README.md]: your call.

Comment (shinnn) [inline README.md]: Thanks, @murphydanger. I'd like to update "CleanCSS constructor" link instead. Can you update the commit?

**Comment by murphydanger** [comment]
@shinnn thoughts?

File: README.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ var minifyCss = require('gulp-minify-css');
 
 ### minifyCss([*options*])
 
-*options*: `Object`
```
Comment (shinnn) [inline README.md]: Needs two spaces here.

Current `master` branch:

image

Yours:

image

Comment (murphydanger) [inline README.md]: that... makes sense.

**Comment by murphydanger** [comment]
Alright, third time's the charm.

**Comment by shinnn** [comment]
:+1:

**Comment by shinnn** [comment]
Ensure the pull request and commits have a useful title in the future.

**Comment by murphydanger** [comment]
Word

**Pull request merged**